### ISSUE TITLE
mcrypt_create_iv is deprecated

### ISSUE THREAD

**Issue by sylfabre** [kunalvarma05/dropbox-php-sdk#107]
As `mcrypt_create_iv` is deprecated, this lib should try to use first OpenSSL in `\Kunnu\Dropbox\Security\RandomStringGeneratorFactory::defaultRandomStringGenerator()`

**Comment by rafamds** [comment]
Any solution?

**Comment by kunalvarma05** [comment]
You can configure the SDK to use OpenSSL by setting the `random_string_generator` to `openssl` in the configuration array and passing that array to the `Dropbox` class.

```php
use Kunnu\Dropbox\Dropbox;
use Kunnu\Dropbox\DropboxApp;

$app = new DropboxApp("client_id", "client_secret");

$config = [
    'random_string_generator' => 'openssl'
];

$dropbox = new Dropbox($app, $config);
```

See: **[Advanced Configuration](https://github.com/kunalvarma05/dropbox-php-sdk/wiki/Advanced-Configuration#random-string-generator)**

MCrypt generator will be removed in the upcoming release.

**Comment by DarkSide666** [comment]
related PR https://github.com/kunalvarma05/dropbox-php-sdk/pull/103

**Comment by kunalvarma05** [comment]
Closed by #103
